index.html

```diff
@@ -598,7 +598,9 @@ <h3>Use Cases and Requirements</h3>
 A conforming document is either a <a>verifiable credential</a> that MUST be
 serialized using the `application/vc+ld+json` media type or a
 <a>verifiable presentation</a> that MUST be serialized using the
-`application/vp+ld+json` media type.
+`application/vp+ld+json` media type. A conforming document MUST be
+secured by at least one securing mechanism as described in Section
+<a href="#securing-mechanisms"></a>.
```

Comment:
By moving this text from the Securing Mechanisms section to the definition of a conforming document, we make it clear that a conforming document MUST be secured. Doing that also enables us to remove a bunch of hand waving we were doing around securing mechanisms.